repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/guidance-main.ftl

```diff
@@ -13,7 +13,9 @@
                     <div class="nhsuk-grid-column-two-thirds">
                         <div class="nhsuk-hero-content">
                             <h1 class="nhsuk-u-margin-bottom-3">${document.title}</h1>
-                            <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">${document.summary}</p>
+                            <#if document.summary??>
+                                <p class="nhsuk-body-l nhsuk-u-margin-bottom-0"><@hst.html formattedText="${document.summary!?replace('\n', '<br>')}"/></p>
```

Comment:
Looks like the summary and the first main content block are quite closer:
![Screenshot 2021-12-09 at 13 25 37](https://user-images.githubusercontent.com/35143542/145404936-498bc172-3626-4d47-b33e-3d9fd55c246b.png)
Probably, we need to get rid of `nhsuk-u-margin-bottom-0` class ?